### PR TITLE
Fix map gen settings

### PR DESCRIPTION
**Pull request by mheguy**

### CODE REVIEW

**Comment by SimonFlapse** [comment]
I can't say much to this. But it looks good to me, I assume you have successfully tested the treat_missing_as_default = false additions. Such was grass, desert and water only.

**Comment by mheguy** [comment]
Water only aka waterworld does not work. I've put in a modding interface request to restore that functionality, but we will have to see how that goes.
As it stands, tetris is the only affected map and it is mentioned in one of the issues.

File: resources/map_gen_settings.lua [hunk 1]
```diff
@@ -255,76 +255,114 @@ return {
...
-            grass = {
-                frequency = 'normal',
-                richness = 'normal',
-                size = 'none'
-            },
-            sand = {
-                frequency = 'normal',
-                richness = 'normal',
-                size = 'none'
+    enable_water = {
```
Comment (SimonFlapse) [inline resources/map_gen_settings.lua:328]: What is the usage of `enable_water` vs `waterworld` https://github.com/Refactorio/RedMew/blob/21a3b9cea724d4b18f07ff05d995aba4b9e42e99/resources/map_gen_settings.lua#L342

**Review by SimonFlapse — APPROVED** [review]
LGTM I don't know if we might need to tweak the preset settings

**Comment by mheguy** [comment]
`waterworld` is/was only water tiles, `enable_water` enables water on, for example, `grass_only` maps.

**Pull request merged**